alibi_detect/cd/sklearn/tests/test_classifier_sklearn.py

```diff
@@ -270,3 +270,90 @@ def test_predict_proba2(model, pred_types, use_calibration, binarize_preds):
+                                                                      (True, True, False)])
+def test_rf_oob(model, p_val, n, n_features, n_folds, preds_type, binarize_preds, use_calibration, use_oob):
+    np.random.seed(0)
+
+    x_ref = np.random.randn(n, n_features)
+    x_test0 = np.random.randn(n, n_features)
+    x_test1 = np.random.randn(n, n_features) + 1
+    cd = ClassifierDriftSklearn(
+        x_ref=x_ref,
+        preds_type=preds_type,
+        model=model,
+        p_val=p_val,
+        n_folds=n_folds,
+        binarize_preds=binarize_preds,
+        use_calibration=use_calibration,
+        use_oob=use_oob
+    )
+
+    preds_0 = cd.predict(x_test0)
+    assert cd.n == len(x_test0) + len(x_ref)
+    assert preds_0['data']['is_drift'] == 0
+    assert preds_0['data']['distance'] >= 0
+
+    preds_1 = cd.predict(x_test1)
+    assert cd.n == len(x_test1) + len(x_test0) + len(x_ref)
+    assert preds_1['data']['is_drift'] == 1
+    assert preds_1['data']['distance'] >= 0
+
+    assert preds_0['data']['distance'] < preds_1['data']['distance']
+    assert cd.meta['params']['preds_type'] == preds_type
+    assert cd.meta['params']['binarize_preds '] == binarize_preds
+
+
+@pytest.mark.parametrize('model', [LogisticRegression(),
+                                   GradientBoostingClassifier(),
+                                   AdaBoostClassifier()])
+@pytest.mark.parametrize('preds_type', ['probs'])
+@pytest.mark.parametrize('use_oob', [True])
+def test_clone_rf1(model, preds_type, use_oob):
+    # should raise an error because `use_oob=True` and the model is different than RF
+    with pytest.raises(ValueError):
+        ClassifierDriftSklearn(x_ref=np.random.randn(100, 5),
+                               model=model,
+                               preds_type=preds_type,
+                               use_oob=use_oob)
+
+
+@pytest.mark.parametrize('model', [RandomForestClassifier()])
+@pytest.mark.parametrize('preds_type', ['probs'])
+@pytest.mark.parametrize('use_calibration', [False, True])
+@pytest.mark.parametrize('use_oob', [True])
+def test_clone_rf2(model, preds_type, use_calibration, use_oob):
+    # this should pass because `use_oob=True` and the model is RF
+    cd = ClassifierDriftSklearn(x_ref=np.random.randn(100, 5),
+                                model=model,
+                                preds_type=preds_type,
+                                use_calibration=use_calibration,
+                                use_oob=use_oob)
+
+    assert cd.model.oob_score
+    assert cd.model.bootstrap
+    assert not cd.use_calibration   # shuld be set to `False` when `use_oob=True`
+
+
+@pytest.mark.parametrize('model', [RandomForestClassifier(oob_score=True),
+                                   RandomForestClassifier(oob_score=False)])
+@pytest.mark.parametrize('preds_type', ['probs'])
+@pytest.mark.parametrize('use_oob', [False])
+def test_clone_rf3(model, preds_type, use_oob):
```

Comment:
please clarify the aim of this test (a comment like in the above tests would do it)